Adyen/Analytics/AnalyticsProvider/TelemetryTracker.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// : nodoc:
+public enum TelemetryFlavor {
```

Comment:
Not sold on the name flavor :D

Reply:
Yeah hahaha "flavor" is the term we're using in telemetry, that's why we're using it. Do you have any suggestion?